omnibus/files/private-chef-cookbooks/private-chef/recipes/redis_lb.rb

```diff
@@ -113,14 +113,14 @@
                       port: redis_data['port'],
                       password: PrivateChef.credentials.get('redis_lb', 'password'))
     xdl = node['private_chef']['lb']['xdl_defaults']
+    xmaint_allowed_ips_list = node['private_chef']['lb']['xmaint_allowed_ips_list']
     banned_ips = PrivateChef['banned_ips']
     maint_mode_ips = PrivateChef['maint_mode_whitelist_ips']
     # Ensure there is no stale data, but first institute
     # a brief maint mode to avoid potential misrouting when
     # we delete old keys.
     redis.hset 'dl_default', '503_mode', true
-    next until redis.spop('banned_ips').nil?
```

Comment:
I would be hesitant in removing this unless we know where it is used or the motivation for adding it.